README.md

```diff
@@ -103,31 +53,29 @@ You can use these fields in the YAML front matter later in your document. For ex
 
 `# {{ page.title }}` would be rendered in HTML as, `<h1>My page title</h1>`.
 
-## Markdown
+### Recomended Markdown Editor
 
-There are dozens of guides and cheatsheets that cover Markdown syntax out there, though this screenshot from the free OS X Markdown editor, [Mou](http://mouapp.com/), is an excellent and concise reference:
-
-![Mou screen shot](http://25.io/mou/img/1.png)
+[Visual Studio Code](https://github.com/Microsoft/vscode) has great support for Scala, Git, and Markdown.
 
 ### Linking to internal pages
 
-The least error-prone way to link between documents, to link to local images, or anything else: `[link text]({{ site.baseurl }}/path/to/page/page.html)`
+The least error-prone way to make links is to use this format: `[link text]({{ site.baseurl }}/path/to/page/page.html)`
 
-Here, `{{ site.baseurl }}` is a site-wide variable that represents the root directory of the static site. So, to display the Scala logo image, located in `img/scala-logo.png`, one must simply write: `![Img alt text]({{ site.baseurl }}/resources/img/scala-logo.png)`
+`{{ site.baseurl }}` is a site-wide variable that represents the root directory of the static site. So, to display the Scala logo image you can simply write: `![Img alt text]({{ site.baseurl }}/resources/img/scala-logo.png)`
 
-## Permalinks
+### Permalinks
```

Comment:
Does anyone know how to fix this rendering issue? It is also present on the master branch.

Reply:
Looks like the character between '##' and 'Permalinks' is not a space.

Reply:
Yup. That fixed it. :-)